src/App.js

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
+import { Routes, Route } from 'react-router-dom';
 import { Container } from 'react-bootstrap';
-import './App.css';
 import Calculator from './components/Calculator';
+import NavBar from './components/Nav';
+import Home from './components/Home';
+import Myself from './components/Myself';
+import './App.css';
 
-const App = () => (
-  <Container>
-    <Calculator />
-  </Container>
-);
+function App() {
+  return (
+    <>
+      <NavBar />
+      <Container>
+        <Routes>
+          <Route path="/mathmagicians" element={<Home />} exact />
+          <Route path="/Calculator" element={<Calculator />} />
+          <Route path="/Myself" element={<Myself />} />
+        </Routes>
```

Comment:
- [ ] It would be nice if your website doesn't have an extra page. Currently, you have an extra route that is empty. You could fix this minor issue just by giving the home page a path of only `/` see it below. 👇 

```suggestion
          <Route path="/" element={<Home />} exact />
          <Route path="/Calculator" element={<Calculator />} />
          <Route path="/Myself" element={<Myself />} />
        </Routes>
```

Remember to change your nav bar also. 

🛠️ 😄

Reply:
Thank you for your suggestions i worked on the requested changes. Thanks again.